### DISCUSSION TITLE
Timezone? Garmin shows activity in the future...

### DISCUSSION THREAD

**Discussion by N6MIK** [philosowaffle/peloton-to-garmin#257] [Q&A] [answered]
Just got things running in Docker on a Synology NAS. First rate stuff, very thankful to the dev and contributors. 
I'm in California, and took a Peloton ride around 5:30PM, however Garmin thinks it happened at midnight:35... tomorrow. 

Any way to adjust the timezone so things line up? Thanks!

**Comment by philosowaffle** [comment, marked as answer]
You can provide Timezone information to the Docker container via the `TZ` environment variable. Checkout the example [docker-compose file in the docs](https://philosowaffle.github.io/peloton-to-garmin/install/docker.html#docker-compose).

**Comment by N6MIK** [comment]
Hello. I changed the default Chigago timezone to Los_Angeles and restarted
the container. New P2G activities are still coming in as GMT-1 Azores time,
Activities coming in from my watch have the correct time in
connect.garmin.com. Is there something else I could be missing? Thank you!

**Reply by sonoranwanderer** [reply]
@N6MIK,  You likely need to change your docker command. Using the `docker-compose.yaml` requires starting docker in the `compose` mode instead of `run` mode. Try `docker compose up` or `docker-compose up` instead of `docker run -v ...` while sitting in the directory with the `configuration.local.json` and `docker-compose.yaml` files.

**Comment by cbordman** [comment]
If you're using 'docker run', add this after all of your -v options..  "-e TZ=America/Chicago" 

for east coast:  America/New_York

**Comment by N6MIK** [comment]
Thanks for all of the replies. As it turns out... this was happening because I had fat-fingers and mounted my *.yaml file into the container as *.yml - so the TZ variable wasn't being picked up at all. Fixed that, and all is good.
